exercises/practice/wordy/.meta/example.cairo

```diff
@@ -7,24 +7,24 @@ enum Operator {
     Invalid,
 }
 
-pub fn answer(word: ByteArray) -> (i32, bool) {
-    let words: Array<ByteArray> = split_string_into_words(word);
+pub fn answer(question: ByteArray) -> i32 {
+    let words: Array<ByteArray> = split_question_into_words(question);
     if words.len() < 3 {
-        return (0, false);
+        panic!("syntax error");
     }
 
     if words[0] != @"What" || words[1] != @"is" {
-        return (0, false);
+        panic!("unknown operation");
     }
 
     let mut result: i32 = match words.get(2) {
```

Comment:
Since you've already ensured that `words` is at least 3 items in length, it's guaranteed that `words.get(2)` will return an element. You can simplify this:
```rust
    let mut result = match parse_int(words[2]) {
        Option::Some(s) => s,
        Option::None => panic!("syntax error"),
    };
```